LabeledIconButton/LabeledIconButton.js

```diff
@@ -130,6 +140,7 @@ const LabeledIconButtonBase = kind({
 					backgroundOpacity={backgroundOpacity}
 					icon={icon}
 					iconComponent={iconComponent}
+					iconOnly
```

Comment:
I thinks issue can be fixed by only this line.

Reply:
The issue could be fixed by this line but I'd like to fix the warning when the story loaded.